src/trace.c

```diff
@@ -1040,24 +1074,48 @@ trace_add_path (const void *parent,
 	if (! path_hash)
 		path_hash = NIH_MUST (nih_hash_string_new (NULL, 2500));
 
-	if (nih_hash_lookup (path_hash, pathname)) {
+	/* Try to insert a new path into hash table,
+	 * or abort in case if it already exists (processed).
+	 */
+	if (! maybe_insert_new_path (path_hash, pathname)) {
 		return 0;
-	} else {
-		NihListEntry *entry;
+	}
```

Comment:
For simple one line if statements. You don't need to put brackets around them. One day I would like to convert this to Linux coding style. I wont enforce that yet, but please, can you remove the brackets. I wasn't going to say anything but I noticed you added unneeded brackets to a one liner below.

Reply:
Thank you!

- Removed unnecessary brackets that contains single return statement.
- Added `Signed-off-by`